### ISSUE TITLE
Force result by environmental value?

### ISSUE THREAD

**Issue by Leont** [Leont/extutils-hascompiler#7]
Should there be an environmental value that forces the answer to this question? If so, what should it be called? `PERL_HASCOMPILER`?

**Comment by toddr** [comment]
Yes that'd be awesome. See also https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=950297

**Comment by haarg** [comment]
This is similar, but the reverse of the EUMM `PUREPERL_ONLY` option.  It would be nice if these could be combined in a sensible way.
